src/features/actorInfo/actions.js

```diff
@@ -0,0 +1,38 @@
+import {
+  ACTOR_LOADED,
+  ACTOR_RESET,
+  ACTORS_LOADED,
+  ACTORS_RESET,
+} from './types';
+
+const currentActorLoaded = (payload) => ({
+  type: ACTOR_LOADED,
+  payload,
+});
+
+export const currentActorReset = () => ({
+  type: ACTOR_RESET,
+});
+
+const actorsLoaded = (payload) => ({
+  type: ACTORS_LOADED,
+  payload,
+});
+
+export const actorsReset = () => ({
+  type: ACTORS_RESET,
+});
+
+export const fetchActorById = (id) => (dispatch, _, api) => {
+  api(`actors/${id}`)
```

Comment:
as an option you can create constants for API urls like you did it for your Route. it's a very common practice